### PR TITLE
Group0- Switch default registry to local harbor

### PR DESCRIPTION
**Pull request by singhshwetaQE**
Issue #4181

Switch the CI default registry to a local harbor instance.

This PR is specifically to test changes made to fix issue 4181

### CODE REVIEW

File: .drone.local.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ pipeline:
       GS_CLIENT_EMAIL: ${GS_CLIENT_EMAIL}
       GS_PRIVATE_KEY: ${GS_PRIVATE_KEY}
       DOMAIN: ${DOMAIN}
+
     commands:
-      - tests/integration-test.sh
-      #- pybot tests/test-cases/Group1-Docker-Commands/1-13-Docker-Version.robot
+     - tests/integration-test.sh
+     #- pybot tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot
```
Comment (chengwang86) [inline .drone.local.yml]: Don't add this file when you commit

**Comment by chengwang86** [comment]
Overall Lgtm but please don't edit the file `.drone.local.yml`.

File: .drone.local.yml (hunk 1 above)
Comment (rogeliosanchez) [inline .drone.local.yml]: Same concern here.  I'll check this out after you've removed that edit.

File: tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot [hunk 2]
```diff
@@ -202,4 +202,3 @@ Docker logs non-existent container
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} logs fakeContainer
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  Error: No such container: fakeContainer
-
```
Comment (chengwang86) [inline tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot]: this file should not be modified in this PR :)

**Pull request merged**